src/Adjudications/AdjudicationDetail/AdjudicationDetail.styles.js

```diff
@@ -1,25 +1,34 @@
 import React from 'react'
 import styled from 'styled-components'
-import { SPACING } from '@govuk-react/constants'
+
+const WrappedText = styled.p`
+  word-break: break-word;
+`
 
 export const LabelAndValue = ({ label, value }) => (
   <>
     <strong className="label">{label}</strong>
-    <p>{value || '--'}</p>
+    <WrappedText>{value || '--'}</WrappedText>
   </>
 )
 
-const WrappedText = styled.p`
-  word-break: break-word;
+const MarginlessPararagraph = styled.p`
+  margin: 0px;
 `
 
-export const IncidentDetails = ({ label, value }) => (
+export const Location = ({ establishment, interiorLocation }) => (
   <>
-    <strong className="label">{label}</strong>
-    <WrappedText>{value || '--'}</WrappedText>
+    <strong className="label">Location</strong>
+    <MarginlessPararagraph>{interiorLocation || '--'}</MarginlessPararagraph>
+    <p>{establishment || '--'}</p>
   </>
 )
 
-export const Sanction = styled.div`
-  padding-bottom: ${SPACING.SCALE_5};
+export const Section = styled.div`
+  border-bottom: 3px solid #f2f2f2;
+  margin-top: 10px;
```

Comment:
`import { spacing } from '@govuk-react/lib';`

`margin-bottom: ${spacing.simple(2)},`

For reference:
```
export const SPACING = {
  SCALE_1: '5px',
  SCALE_2: '10px',
  SCALE_3: '15px',
  SCALE_4: '20px',
  SCALE_5: '30px',
  SCALE_6: '60px',
};
```

Reply:
Just to note, `spacing.simple()` means it will be that value regardless of screen size.

If you ever wanted a responsive value you can use helpers like:
`spacing.responsiveMargin()`
or
`spacing.withWhiteSpace`

That ray will adjust the margin at the appropriate breakpoints. [Reference](https://github.com/alphagov/govuk-frontend/blob/master/src/settings/_spacing.scss)

Reply:
cool - I meant to replace this with the constants but missed it. 
The helpers seem much better so I've replaced this now